docs/ECO-METADATA.md

```diff
@@ -28,7 +28,7 @@ Note that there is no need to specify the application instance identity since th
 
 curl <http://169.254.169.254/eve/v1/network.json>
 
-{"caller-ip":"10.1.0.2:39380","external-ipv4":"192.168.1.10","hostname":"afa43e51-56b7-4021-a5fa-4272b0381913"}
+{"caller-ip":"10.1.0.2:39380","external-ipv4":"192.168.1.10","hostname":"afa43e51-56b7-4021-a5fa-4272b0381913", "app-instance-uuid":"afa43e51-56b7-4021-a5fa-4272b0381913","device-uuid":"6c6cb828-b0be-4166-9daf-ab353430dbc1","device-name":"test-system","project-name":"test-project","project-uuid":"4d12659e-6c9f-44bc-8e98-d525347afae6","enterprise-name":"testing","enterprise-uuid":"AAFlABBtJ0mP_lhJjIyVzjzgMXiR"}
```

Comment:
Is the enterprise thing not a uuid format but a string as in this example? If so we should perhaps rename it to enterprise-id.

Reply:
Yep enterprise is not uuid. I will change it to id.